### PR TITLE
[release-1.27] Adresses CVE-2024-1753 and CVE-2024-24786, bump to V1.27.4

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Try 2.  

Replaces #5429

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
The CI is dead in this repo and we only have about two months of required support for it.  After discussing with Matt, we've decided to move forward with this, as long as the equivalent PR #5434 from release-1.29 passed tests.  We'll then vendor into Podman, ensure CI works there, then hand off to QE.

@mheon @nalind @rhatdan PTAL and merge

**Comment by mheon** [comment]
LGTM

File: tests/conformance/conformance_test.go [hunk 1]
```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/containers/image/v5/transports"
 	"github.com/containers/image/v5/types"
 	"github.com/containers/storage"
+
```
Comment (nalind) [inline tests/conformance/conformance_test.go:35]: Not a blocker, but if we have to revise this PR for any reason, this blank line doesn't belong here.

**Review by giuseppe — APPROVED** [review]
/lgtm

**Comment by nalind** [comment]
We turned off CI for this branch, so I don't expect Total Success to ever be reported.
/override "Total Success"

File: tests/conformance/conformance_test.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply tests/conformance/conformance_test.go:35]: DOH, just saw this comment have inadvertently added it to a number of branches.  Will fix what I can without going nuts.

**Pull request merged**